src/courtroom.cpp

```diff
@@ -1597,13 +1596,11 @@ void Courtroom::on_chat_return_pressed()
   int f_emote_mod = ao_app->get_emote_mod(current_char, current_emote);
 
   // needed or else legacy won't understand what we're saying
-  if (objection_state > 0) {
-    if (ui_pre->isChecked()) {
-      if (f_emote_mod == 4 || f_emote_mod == 5)
-        f_emote_mod = 6;
-      else
-        f_emote_mod = 2;
-    }
+  if (objection_state > 0 && ui_pre->isChecked()) {
+    if (f_emote_mod == 4 || f_emote_mod == 5)
+      f_emote_mod = 6;
+    else
+      f_emote_mod = 2;
```

Comment:
wait does the logic actually change in any way, isn't it just condensing an if check but the inherent logic is the same? Has this been tested to work?

Reply:
I honestly don't remember what I changed here to make this work, but I just tested it and it _does_ work, so make of that what you will